src/tests/bezier-subdivision-test.cpp

```diff
@@ -0,0 +1,363 @@
+/** @file
+ * @brief Unit tests for Affine
+ * Uses the Google Testing Framework
+ *//*
+ * Authors:
+ *   Alexander Brock <Brock.Alexander@web.de>
+ *
+ * Copyright 2010 Authors
```

Comment:
Because I copied it from somewhere else and didn't pay enough attention, I changed it, thank you.